13. Select_blog_comment.md

```diff
@@ -0,0 +1,12 @@
+13. Lấy 5 blog mới nhất và số lượng comment cho từng blog
+
+```mysql
+
+SELECT  blog.id, blog.title,count(target_id), blog.updated_at  FROM blog.blog
+INNER JOIN blog.comment
+ON blog.id = comment.target_id 
```

Comment:
Sử dụng
LEFT JOIN comment ON (comment.target_table = 'blog AND comment.target_id = blog.id)
Thay cho điều kiện WHERE ở dưới